providers/implementations/asymciphers/sm2_enc.c

```diff
@@ -0,0 +1,226 @@
+/*
+ * Copyright 2020 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+
+#include <openssl/crypto.h>
+#include <openssl/evp.h>
+#include <openssl/core_dispatch.h>
+#include <openssl/core_names.h>
+#include <openssl/params.h>
+#include <openssl/err.h>
+#include <crypto/sm2.h>
```

Comment:
Travis is having issues with the include here I think.

Reply:
This was in the event of a no-ec or no-sm2 build. Down to a missing "$" sign in the build.info. Fixed now.